### PR TITLE
Adding Analysis Service Dataplane module to AzureRm rollup module

### PR DESCRIPTION
**Pull request by shahabhijeet**

### CODE REVIEW

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

File: tools/AzureRM/AzureRM.psd1 [hunk 1]
```diff
@@ -54,6 +54,7 @@ CLRVersion = '4.0'
 RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '2.4.0'; }, 
                @{ModuleName = 'Azure.Storage'; RequiredVersion = '2.4.0'; }, 
                @{ModuleName = 'AzureRM.AnalysisServices'; RequiredVersion = '0.0.2'}, 
+			   @{ModuleName = 'Azure.AnalysisServices'; RequiredVersion = '0.0.1'},
```
Comment (cormacpayne) [inline tools/AzureRM/AzureRM.psd1]: @shahabhijeet can you fix the indentation for this?

File: tools/PublishModules.ps1 [hunk 2]
```diff
@@ -61,6 +61,8 @@ function Get-TargetModules
               $targets += $module.FullName
             }
           }
+		  $targets += "$resourceManagerRootFolder\Azure.AnalysisServices"
```
Comment (cormacpayne) [inline tools/PublishModules.ps1]: @shahabhijeet same comment about indentation

File: tools/AzureRM/AzureRM.psd1 (hunk 1 above)
Comment (markcowl) [inline tools/AzureRM/AzureRM.psd1]: +1

**Comment by markcowl** [comment]
LGTM once the indentation is fixed.

**Comment by shahabhijeet** [comment]
@markcowl @cormacpayne it was odd that, the local file that I had do not show these misalignments. So I had to edit the file on GitHub to fix these alignment issues.

**Pull request merged**